src/main/java/com/fakng/fakngagrgtr/parser/ApiParser.java

```diff
@@ -5,30 +5,39 @@
 import org.springframework.stereotype.Component;
 import org.springframework.web.reactive.function.client.WebClient;
 import org.springframework.web.reactive.function.client.WebClient.ResponseSpec;
+import org.springframework.web.util.UriBuilder;
 import reactor.core.publisher.Mono;
 
+import java.net.URI;
+import java.util.function.Function;
+
 @Component
 @Slf4j
 public abstract class ApiParser extends Parser {
```

Comment:
It seems to me, or does this class look more like a parser configuration? It creates beans and has almost no business logic.

Maybe it should be renamed and make methods static?